src/graphql-requests/queries.tsx

```diff
@@ -1 +1,25 @@
-import { gql } from "apollo-boost";
+import gql from "graphql-tag";
+
+export const GET_USER = gql`
+   {
+      user(id: id) {
+         id
+         username
+         email
+         first_name
+         last_name
+      }
+   }
+`;
+
```

Comment:
I asked Ray if he could create us a mutation to look for users by email, so we can add a GQL query which takes email as an argument as well.

Reply:
Awesome that would be very helpful on the FE!